lib/widgets/pin_code_widget.dart

```diff
@@ -27,9 +34,24 @@ class PinCodeWidgetState extends State<PinCodeWidget> {
     super.initState();
     _enteredPinCode = "";
     _errorMessage = "";
+    if (widget.onFingerprintEntered != null) {
+      widget.userProfileBloc.userStream.firstWhere((u) => u.securityModel.enrolledBiometrics != "").then((_) {
+        _validateBiometrics();
```

Comment:
Look like this is triggered on init and not cancelled. I can see some weird behavior when someone open this screen, then close it and the biometric validation is still in progress...
Later on it could trigger a flush bar on error or something unexpected.